src/windows/wsladiag/wsladiag.cpp

```diff
@@ -329,6 +326,11 @@ int wmain(int, wchar_t**)
     catch (...)
     {
         const auto hr = wil::ResultFromCaughtException();
+        if (hr == E_INVALIDARG)
```

Comment:
`E_INVALIDARG` can be thrown for other reasons for I would not recommend having a specific catch block here. 

Once we have ExecutionContext based errors, we can use those to show command line parsing specific errors